test/e2e_node/remote/node_e2e.go

```diff
@@ -66,29 +67,41 @@ func (n *NodeE2ERemote) SetupTestPackage(tardir string) error {
 	}
 
 	// Include the GCI/COS mounter artifacts in the deployed tarball
+	err = tarAddCOSMounter(tardir)
+	if err != nil {
+		return err
+	}
+	return nil
+}
+
+func tarAddFile(tardir, source, destdir, destfile string) error {
```

Comment:
It seems that this is only used to copy one file, right?

If so, I think we could just path the full path of the file in, and use [`filepath.Dir`](https://golang.org/pkg/path/filepath/#Dir) to get directory name, and [`filepath.Base`](https://golang.org/pkg/path/filepath/#Base) to get the file name.

This will make this function a little bit cleaner.